x-pack/plugins/profiling/public/plugin.tsx

```diff
@@ -5,17 +5,66 @@
  * 2.0.
  */
 
-import { AppMountParameters, CoreSetup, CoreStart, Plugin } from '@kbn/core/public';
+import {
+  AppMountParameters,
+  CoreSetup,
+  CoreStart,
+  DEFAULT_APP_CATEGORIES,
+  Plugin,
+} from '@kbn/core/public';
+import { i18n } from '@kbn/i18n';
+import { from, map } from 'rxjs';
+import type { NavigationSection } from '@kbn/observability-plugin/public';
 import { getServices } from './services';
+import type { ProfilingPluginPublicSetupDeps } from './types';
 
-export class ProdfilerPlugin implements Plugin {
-  public setup(core: CoreSetup) {
+export class ProfilingPlugin implements Plugin {
+  public setup(core: CoreSetup, deps: ProfilingPluginPublicSetupDeps) {
     // Register an application into the side navigation menu
+
+    const links = [
+      {
+        id: 'flamegraphs',
+        title: i18n.translate('xpack.profiling.navigation.flamegraphLinkLabel', {
+          defaultMessage: 'Flamegraphs',
+        }),
+        path: '/',
+      },
+    ];
+
+    const section$ = from(core.getStartServices()).pipe(
+      map(([coreStart, pluginsStart]) => {
+        const sections: NavigationSection[] = [
+          {
+            label: i18n.translate('xpack.profiling.navigation.sectionLabel', {
+              defaultMessage: 'Profiling',
+            }),
+            entries: links.map((link) => {
+              return {
+                app: 'profiling',
+                label: link.title,
+                path: link.path,
+              };
+            }),
+            sortKey: 700,
```

Comment:
This is a bit of a magic number to make it appear at the bottom. We use increments of 100, 600 is from the second-to-last link.